### ISSUE TITLE
Error in blz.compute(t.weight.max()) when upgrading blaze/odo

### ISSUE THREAD

**Issue by chdoig** [ContinuumIO/blaze#995]
I'm getting this error when I updated my dependencies to:

blaze                     0.7.2                np19py27_0  
odo                       0.3.1                np19py27_0  

Code:
```
 t = blz.Data(input_file)
 MAX =  blz.compute(t.weight.max())
```

I can send you the input file if you need it, @mrocklin 

Error:
```bash
---> 37         MAX =  blz.compute(t.weight.max())
     38         MIN = blz.compute(t.weight.min())
     39 

/lib/python2.7/site-packages/multipledispatch/dispatcher.pyc in __call__(self, *args, **kwargs)
    161             self._cache[types] = func
    162         try:
--> 163             return func(*args, **kwargs)
    164 
    165         except MDNotImplementedError:

/lib/python2.7/site-packages/blaze/interactive.pyc in compute(expr, **kwargs)
    160         raise ValueError("No data resources found")
    161     else:
--> 162         return compute(expr, resources, **kwargs)
    163 
    164 

/lib/python2.7/site-packages/multipledispatch/dispatcher.pyc in __call__(self, *args, **kwargs)
    161             self._cache[types] = func
    162         try:
--> 163             return func(*args, **kwargs)
    164 
    165         except MDNotImplementedError:

/lib/python2.7/site-packages/blaze/compute/core.pyc in compute(expr, d, **kwargs)
    452         d3 = dict([(e, pre_compute_(expr2, dat, **kwargs))
    453                         for e, dat in d2.items()
--> 454                         if e in expr2])
    455     else:
    456         d3 = d2

/lib/python2.7/site-packages/multipledispatch/dispatcher.pyc in __call__(self, *args, **kwargs)
    161             self._cache[types] = func
    162         try:
--> 163             return func(*args, **kwargs)
    164 
    165         except MDNotImplementedError:

/lib/python2.7/site-packages/blaze/compute/csv.pyc in pre_compute(expr, data, comfortable_memory, chunksize, **kwargs)
     48         return into(chunks(pd.DataFrame), data, dshape=leaf.dshape, **kwargs)
     49     else:
---> 50         return into(pd.DataFrame, data, dshape=leaf.dshape, **kwargs)
     51 
     52 

/lib/python2.7/site-packages/multipledispatch/dispatcher.pyc in __call__(self, *args, **kwargs)
    161             self._cache[types] = func
    162         try:
--> 163             return func(*args, **kwargs)
    164 
    165         except MDNotImplementedError:

/lib/python2.7/site-packages/odo/into.pyc in into_type(a, b, **kwargs)
     23         if 'dshape' not in kwargs:
     24             kwargs['dshape'] = discover(b)
---> 25     return convert(a, b, **kwargs)
     26 
     27 

/lib/python2.7/site-packages/odo/core.pyc in __call__(self, *args, **kwargs)
     28 
     29     def __call__(self, *args, **kwargs):
---> 30         return _transform(self.graph, *args, **kwargs)
     31 
     32 

/lib/python2.7/site-packages/odo/core.pyc in _transform(graph, target, source, excluded_edges, ooc_types, **kwargs)
     41     pth = path(graph, type(source), target,
     42                excluded_edges=excluded_edges,
---> 43                ooc_types=ooc_types)
     44     try:
     45         for (A, B, f) in pth:

/lib/python2.7/site-packages/odo/core.pyc in path(graph, source, target, excluded_edges, ooc_types)
     77                                     if issubclass(n, oocs)])
     78     with without_edges(graph, excluded_edges) as g:
---> 79         pth = nx.shortest_path(g, source=source, target=target, weight='cost')
     80         result = [(src, tgt, graph.edge[src][tgt]['func'])
     81                   for src, tgt in zip(pth, pth[1:])]

/lib/python2.7/site-packages/networkx/algorithms/shortest_paths/generic.pyc in shortest_path(G, source, target, weight)
    136                 paths=nx.bidirectional_shortest_path(G,source,target)
    137             else:
--> 138                 paths=nx.dijkstra_path(G,source,target,weight)
    139 
    140     return paths

/lib/python2.7/site-packages/networkx/algorithms/shortest_paths/weighted.pyc in dijkstra_path(G, source, target, weight)
     75     """
     76     (length, path) = single_source_dijkstra(G, source, target=target,
---> 77                                             weight=weight)
     78     try:
     79         return path[target]

/lib/python2.7/site-packages/networkx/algorithms/shortest_paths/weighted.pyc in single_source_dijkstra(G, source, target, cutoff, weight)
    337                 edata.append((w, {weight: minweight}))
    338         else:
--> 339             edata = iter(G[v].items())
    340 
    341         for w, edgedata in edata:

/lib/python2.7/site-packages/networkx/classes/graph.pyc in __getitem__(self, n)
    317         {1: {}}
    318         """
--> 319         return self.adj[n]
    320 
    321 

KeyError: <class 'into.backends.csv.CSV'>

```

**Comment by chdoig** [comment]
Just read the email exchange about odo and blaze. 

Got fixed when I upgraded to 0.7.3.

**Comment by cpcloud** [comment]
Great! Closing.
